ciecplib/sessions.py

```diff
@@ -45,6 +45,7 @@ def __init__(
             username=None,
             password=None,
             cookiejar=None,
+            **kwargs
```

Comment:
Please add trailing commas because I'm pedantic

```suggestion
            **kwargs,
```

Reply:
Even though there should not be anything after `**kwargs`, ever?